Sources/StyleguideV2/HTML/HTMLPrinter.swift

```diff
@@ -31,19 +31,25 @@ public struct HTMLPrinter {
       }
       for (className, style) in styles {
         sheet.append(currentIndentation)
-        sheet.append("\(className){\(style)}")
+        if configuration.forceImportant {
+          sheet.append("\(className){\(style) !important}")
+        } else {
+          sheet.append("\(className){\(style)}")
+        }
         sheet.append(configuration.newline)
       }
     }
     return sheet
   }
 
   public struct Configuration {
+    let forceImportant: Bool
```

Comment:
We could consider calling this `isHTMLEmail` or `isEmail` instead to document why, but we also know we wanna reconsider the HTML library design with more use and after the dust settles.